torch/csrc/lazy/core/lazy_graph_executor.cpp

```diff
@@ -403,6 +403,40 @@ LazyGraphExecutor* LazyGraphExecutor::Get() {
   return executor;
 }
 
+void LazyGraphExecutor::InitExecutionThread() {
+  auto func = [this]() {
+    std::function<void()> exec;
+    while (execution_queue_->Pop(&exec)) {
+      try {
+        exec();
+      } catch (const std::exception& ex) {
+        LOG(ERROR) << "Exception from running thread pool closure: "
```

Comment:
shouldn't we throw the error here?

Reply:
I guess my question is will  `LOG(ERROR)` actually error out(if not, should we ?) or just print a log message